### ISSUE TITLE
ResNet state_dict contains keys not used by model

### ISSUE THREAD

**Issue by s0tt** [facebookresearch/detectron2#4108]
## Instructions To Reproduce the Issue:

1. Full runnable code or full changes you made:
```
def setup(path):
    """
    Create configs and perform basic setups.
    """
    cfg = get_cfg()
    cfg.merge_from_file(path)
    cfg.freeze()
    set_global_cfg(cfg)
    return cfg

```
The R-50.pkl file I manually downloaded from: https://dl.fbaipublicfiles.com/detectron/ImageNetPretrained/MSRA/R-50.pkl

mask_rcnnR_50_FPN_1x.yaml (just the default config):
```
_BASE_: "../Base-RCNN-FPN.yaml"
MODEL:
  WEIGHTS: "detectron2://ImageNetPretrained/MSRA/R-50.pkl"
  MASK_ON: True
  RESNETS:
    DEPTH: 50
  ROI_HEADS:
    NUM_CLASSES: 1203
    SCORE_THRESH_TEST: 0.0001
INPUT:
  MIN_SIZE_TRAIN: (640, 672, 704, 736, 768, 800)
DATASETS:
  TRAIN: ("lvis_v1_train",)
  TEST: ("lvis_v1_val",)
TEST:
  DETECTIONS_PER_IMAGE: 300  # LVIS allows up to 300
SOLVER:
  STEPS: (120000, 160000)
  MAX_ITER: 180000  # 180000 * 16 / 100000 ~ 28.8 epochs
DATALOADER:
  SAMPLER_TRAIN: "RepeatFactorTrainingSampler"
  REPEAT_THRESHOLD: 0.001
```
2. What exact command you run:
```
cfg = setup("detectron2/configs/LVISv1-InstanceSegmentation/mask_rcnn_R_50_FPN_1x.yaml")
model = DefaultTrainer.build_model(cfg)
modelCkpt = DetectionCheckpointer(model).load("./detectron2/R-50.pkl")

```

3. __Full logs__ or other relevant observations:
```
Some model parameters or buffers are not found in the checkpoint:
Some model parameters or buffers are not found in the checkpoint:
[34mbackbone.fpn_lateral2.{bias, weight}[0m
[34mbackbone.fpn_lateral3.{bias, weight}[0m
[34mbackbone.fpn_lateral4.{bias, weight}[0m
[34mbackbone.fpn_lateral5.{bias, weight}[0m
[34mbackbone.fpn_output2.{bias, weight}[0m
[34mbackbone.fpn_output3.{bias, weight}[0m
[34mbackbone.fpn_output4.{bias, weight}[0m
[34mbackbone.fpn_output5.{bias, weight}[0m
[34mproposal_generator.rpn_head.anchor_deltas.{bias, weight}[0m
[34mproposal_generator.rpn_head.conv.{bias, weight}[0m
[34mproposal_generator.rpn_head.objectness_logits.{bias, weight}[0m
[34mroi_heads.box_head.fc1.{bias, weight}[0m
[34mroi_heads.box_head.fc2.{bias, weight}[0m
[34mroi_heads.box_predictor.bbox_pred.{bias, weight}[0m
[34mroi_heads.box_predictor.cls_score.{bias, weight}[0m
[34mroi_heads.mask_head.deconv.{bias, weight}[0m
[34mroi_heads.mask_head.mask_fcn1.{bias, weight}[0m
[34mroi_heads.mask_head.mask_fcn2.{bias, weight}[0m
[34mroi_heads.mask_head.mask_fcn3.{bias, weight}[0m
[34mroi_heads.mask_head.mask_fcn4.{bias, weight}[0m
[34mroi_heads.mask_head.predictor.{bias, weight}[0m
The checkpoint state_dict contains keys that are not used by the model:
  [35mfc1000.{bias, weight}[0m
  [35mres2.0.conv1.bias[0m
  [35mres2.0.conv2.bias[0m
  [35mres2.0.conv3.bias[0m
  [35mres2.0.shortcut.bias[0m
  [35mres2.1.conv1.bias[0m
  [35mres2.1.conv2.bias[0m
  [35mres2.1.conv3.bias[0m
  [35mres2.2.conv1.bias[0m
  [35mres2.2.conv2.bias[0m
  [35mres2.2.conv3.bias[0m
  [35mres3.0.conv1.bias[0m
  [35mres3.0.conv2.bias[0m
  [35mres3.0.conv3.bias[0m
  [35mres3.0.shortcut.bias[0m
  [35mres3.1.conv1.bias[0m
  [35mres3.1.conv2.bias[0m
  [35mres3.1.conv3.bias[0m
  [35mres3.2.conv1.bias[0m
  [35mres3.2.conv2.bias[0m
  [35mres3.2.conv3.bias[0m
  [35mres3.3.conv1.bias[0m
  [35mres3.3.conv2.bias[0m
  [35mres3.3.conv3.bias[0m
  [35mres4.0.conv1.bias[0m
  [35mres4.0.conv2.bias[0m
  [35mres4.0.conv3.bias[0m
  [35mres4.0.shortcut.bias[0m
  [35mres4.1.conv1.bias[0m
  [35mres4.1.conv2.bias[0m
  [35mres4.1.conv3.bias[0m
  [35mres4.2.conv1.bias[0m
  [35mres4.2.conv2.bias[0m
  [35mres4.2.conv3.bias[0m
  [35mres4.3.conv1.bias[0m
  [35mres4.3.conv2.bias[0m
  [35mres4.3.conv3.bias[0m
  [35mres4.4.conv1.bias[0m
  [35mres4.4.conv2.bias[0m
  [35mres4.4.conv3.bias[0m
  [35mres4.5.conv1.bias[0m
  [35mres4.5.conv2.bias[0m
  [35mres4.5.conv3.bias[0m
  [35mres5.0.conv1.bias[0m
  [35mres5.0.conv2.bias[0m
  [35mres5.0.conv3.bias[0m
  [35mres5.0.shortcut.bias[0m
  [35mres5.1.conv1.bias[0m
  [35mres5.1.conv2.bias[0m
  [35mres5.1.conv3.bias[0m
  [35mres5.2.conv1.bias[0m
  [35mres5.2.conv2.bias[0m
  [35mres5.2.conv3.bias[0m
  [35mstem.conv1.bias[0m
```

## Expected behavior:
Expected to use all parameters from pre-trained resnet, not leaving out the bias of all the convs.
Am i missing something?
Thanks for your help!

**Comment by an99990** [comment]
i have the same issue, did you find anything ?

**Comment by s0tt** [comment]
No, but i now believe this is expected behaviour as the results from training are reasonable for me.

**Comment by ppwwyyxx** [comment]
This is expected: the bias are useless because resnet has batchnorm after all convolutions.
